avocado/core/data_dir.py

```diff
@@ -154,40 +154,40 @@ def get_logs_dir():
                        SYSTEM_LOG_DIR, USER_LOG_DIR)
 
 
-def create_job_logs_dir(logdir=None, unique_id=None):
+def create_job_logs_dir(base_dir=None, unique_id=None):
     """
     Create a log directory for a job, or a stand alone execution of a test.
 
-    :param logdir: Base log directory, if `None`, use value from configuration.
+    :param base_dir: Base log directory, if `None`, use value from configuration.
```

Comment:
Yep, this name suits better the purpose. I'm a bit afraid it might break custom plugins, do you think we should allow to fall-back to `logdir` in case `args.base_logdir` does not exists at least for let's say 3 sprints?

Reply:
Everything is possible, but I don't think this is an API that is going to be used by the average plugin out there.  One other point is that this would only affect use cases where the parameters names are used.

But, in the end, I don't think we should add this kind of workaround code, since the safeguards for stability of the public interfaces are already there, namely the LTS release.

Reply:
OK